tests/test_session.py

```diff
@@ -304,15 +304,20 @@ def test_solverworkflow_in_solver_session(new_solver_session):
 
 @pytest.mark.dev
 @pytest.mark.fluent_232
-def test_read_case_using_light_io(new_solver_session):
+@pytest.mark.skip("Failing in github")
+def test_read_case_using_light_io(with_launching_container):
     import_filename = examples.download_file(
         "mixing_elbow.cas.h5", "pyfluent/mixing_elbow"
     )
-    pyfluent.USE_LIGHT_IO = True
-    new_solver_session.read_case(file_name=import_filename)
-    new_solver_session.setup.models.energy.enabled = False
-    old_fluent_connection_id = id(new_solver_session.fluent_connection)
-    while id(new_solver_session.fluent_connection) == old_fluent_connection_id:
+    solver = pyfluent.launch_fluent(case_filepath=import_filename, use_light_io=True)
```

Comment:
@mkundu1 I'm wondering if we could rename `use_light_io` to simply `lightweight_mode` to de-emphasise the IO aspect?